modules/orionode.debug.server/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "orion-debugger-server",
+  "version": "0.1.0",
+  "main": "server.js",
+  "description": "A debugger server for orion using VS Code debugger protocol.",
+  "keywords": [],
+  "author": "Orion contributors <orionnode@gmail.com>",
+  "license": "Eclipse Public License + Eclipse Distribution License",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "dependencies": {
+    "express": "^4.14.1",
+    "socket.io": "^1.7.2",
+    "vscode-debugadapter": "^1.16.0",
+    "vscode-debugprotocol": "^1.16.0"
+  },
+  "devDependencies": {},
+  "engines": {
+    "node": "^4.0.0"
+  },
+  "scripts": {
+    "start": "node server.js",
+    "install": "node installAdapters.js",
+    "demo": "node demo/demoServer.js"
```

Comment:
It will run on port 3000. It's a demo for the VS Code debug protocol. Basically you can write and send any debug request in JSON by your self, just like using a CLI debugger.